### PR TITLE
Remove unused executor

### PR DESCRIPTION
**Pull request by mpolden**
@jonmv

### CODE REVIEW

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/DiskReplacer.java [hunk 1]
```diff
@@ -39,10 +34,13 @@ public class DiskReplacer extends NodeRepositoryMaintainer {
 
     @Override
     protected double maintain() {
-        NodeList nodes = nodeRepository().nodes().list().rebuilding(true);
+        NodeList candidates = nodeRepository().nodes().list().rebuilding(true);
+        if (candidates.isEmpty()) {
+            return 0;
```
Comment (jonmv) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/DiskReplacer.java:39]: Not 1?

**Review by jonmv — APPROVED** [review]
Consider return value when no work to do.

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/DiskReplacer.java (hunk 1 above)
Comment (mpolden) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/DiskReplacer.java:39]: The definition has changed. 0 now indicates there is no work to do.

**Comment by jonmv** [comment]
How do we then differentiate between complete failure, and no work to do?

**Pull request merged**